Combinatorial_map/package_info/Combinatorial_map/dependencies

```diff
@@ -14,8 +17,15 @@ Intersections_3
 Interval_support
 Kernel_23
 Kernel_d
+Linear_cell_complex
+Modifier
 Modular_arithmetic
 Number_types
+Polyhedron
+Polyhedron_IO
 Profiling_tools
+Property_map
 STL_Extension
 Stream_support
+Surface_mesh
+Surface_mesh_topology
```

Comment:
What happended?! Why are there suddenly so many new dependencies? We cannot accept that because `Combinatorial_map` is licensed LGPLv+3, and it cannot depend on GPL package.

Reply:
I don't know; this is the result of the make packages_dependencies !!
Maybe this is due to some links in the doc ?